analyzer/src/main/kotlin/managers/MavenDependencyHandler.kt

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.managers
+
+import org.apache.maven.project.MavenProject
+import org.apache.maven.project.ProjectBuildingException
+
+import org.eclipse.aether.graph.DependencyNode
+
+import org.ossreviewtoolkit.analyzer.managers.utils.DependencyHandler
+import org.ossreviewtoolkit.analyzer.managers.utils.MavenSupport
+import org.ossreviewtoolkit.analyzer.managers.utils.identifier
+import org.ossreviewtoolkit.model.OrtIssue
+import org.ossreviewtoolkit.model.Package
+import org.ossreviewtoolkit.model.PackageLinkage
+import org.ossreviewtoolkit.model.createAndLogIssue
+import org.ossreviewtoolkit.utils.collectMessagesAsString
+import org.ossreviewtoolkit.utils.log
+import org.ossreviewtoolkit.utils.showStackTrace
+
+/**
+ * A specialized [DependencyHandler] implementation for the dependency model of Maven.
+ */
+class MavenDependencyHandler(
+    /** The name of the associated package manager. */
+    val managerName: String,
+
+    /** The helper object to invoke Maven-related functionality. */
+    val mvn: MavenSupport,
```

Comment:
Just some thoughts: `mvn` reminds me of the CLI, and "maven" is already in the class names, so how about naming this "support" instead?